File: Codewars/8kyu/8kyu_reversed_words/src/main.py
import unittest

def reverseWords(str):
    # return ' '.join(reversed(str.split(' ')))
    # return ' '.join(reversed(str.split()))
    return ' '.join(str.split(' ')[::-1])


class Test(unittest.TestCase):
    def test_1(self):
        self.assertEqual(reverseWords("hello world!"), "world! hello")

if __name__ == '__main__':
    unittest.main()

File: Codewars/8kyu/8kyu_reversed_words/src/main.js
const assert = require('assert')

function mysolution(str) {
    return str.split(' ').reverse().join(' ')
}

const reverseWords = mysolution

describe("reverseWords",function(){
    it("should work for some examples", function(){
        assert.equal(reverseWords("hello world!"), "world! hello")
        assert.equal(reverseWords("yoda doesn't speak like this" ),  "this like speak doesn't yoda")
        assert.equal(reverseWords("foobar"                       ),  "foobar")
        assert.equal(reverseWords("kata editor"                  ),  "editor kata")
        assert.equal(reverseWords("row row row your boat"        ),  "boat your row row row")
    });
});